sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/responses/AzureResponsesTest.java

```diff
@@ -144,6 +147,48 @@ public void listInputItems(HttpClient httpClient, AzureResponsesServiceVersion s
         }
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.responses.TestUtils#getTestParametersResponses")
+    public void testResponsesInputContentImageBase64(HttpClient httpClient, AzureResponsesServiceVersion serviceVersion) {
+        ResponsesClient client = getAzureResponseClient(httpClient, serviceVersion);
+
+        try {
+            openImageFileBase64Runner((base64Image) -> {
+                CreateResponsesRequest request = new CreateResponsesRequest(CreateResponsesRequestModel.GPT_4O_MINI,
+                    Arrays.asList(
+                        new ResponsesSystemMessage(Arrays.asList(new ResponsesInputContentText(
+                            "You are a helpful assistant that describes images"))),
+                        new ResponsesUserMessage(Arrays.asList(
+                            new ResponsesInputContentText("Please describe this image"),
+                            new ResponsesInputContentImage().setImageUrl(
+                                "data:image/jpeg;base64," + base64Image)))));
+
+                ResponsesResponse response = client.createResponse(request);
+                assertImageResponseForAzure(response);
+
+            });
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.responses.TestUtils#getTestParametersResponses")
+    public void testResponsesInputContentImageUrl(HttpClient httpClient, AzureResponsesServiceVersion serviceVersion) {
+        ResponsesClient client = getAzureResponseClient(httpClient, serviceVersion);
+
+        CreateResponsesRequest request = new CreateResponsesRequest(CreateResponsesRequestModel.GPT_4O_MINI,
+            Arrays.asList(
+                new ResponsesSystemMessage(Arrays.asList(new ResponsesInputContentText(
+                    "You are a helpful assistant that describes images"))),
+                new ResponsesUserMessage(Arrays.asList(
+                    new ResponsesInputContentText("Please describe this image"),
+                    new ResponsesInputContentImage().setImageUrl("https://upload.wikimedia.org/wikipedia/commons/thumb/4/44/Microsoft_logo.svg/512px-Microsoft_logo.svg.png")))));
+
+        ResponsesResponse response = client.createResponse(request);
+        assertImageResponseForAzure(response);
+    }
+
```

Comment:
Both tests are great! Once we've locked down on the implementation we will need to write the async  version and the `NonAzure` versions too.